modules/gapi/include/opencv2/gapi/media.hpp

```diff
@@ -30,6 +31,8 @@ class GAPI_EXPORTS MediaFrame {
     View access(Access) const;
     cv::GFrameDesc desc() const;
 
+    cv::util::any blobParams() const;
```

Comment:
yeah but at the same time there params have no any other use right now.. not sure if it deserves to be named in such a generic way

Reply:
I would left it as `blobParams` for now. Technically, yes, it could be a generic method differently utilized, but currently it is only used to get IE parameters from frame.